nvflare/fuel/hci/client/static_service_finder.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .api_spec import ServiceFinder
+
+
+class StaticServiceFinder(ServiceFinder):
```

Comment:
user should be using DummyOverseerAgent + ServiceFinderByOverseer for static service?

Reply:
Will address this in another PR.